README.md

```diff
@@ -41,21 +41,24 @@ When building packages that use CUDA, a tar package of TensorRT for the intended
 
 ### Installing the Open-CE Build Tools
 
-To get the Open-CE build tools, one can either clone the source code from [github](https://github.com/open-ce/open-ce-builder) or install it via `pip` as below - 
+To get the Open-CE build tools, one can either install it via `conda`, via `pip` or clone the source code from [github](https://github.com/open-ce/open-ce-builder) as below - 
 
+```bash
+# Conda install from the open-ce channel
+conda install -c open-ce open-ce
```

Comment:
Should the channel and package here have -builder?
At least the package should probably.

Reply:
Yep, thanks!